util/easing/pkg.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+pkg.name: util/easing
+pkg.description: Library containing easing functions
+pkg.author: "Apache Mynewt <dev@mynewt.apache.org>"
+pkg.homepage: "http://mynewt.apache.org/"
+pkg.keywords:
+    - easing
+
+pkg.cflags:
+    - "-lm"
```

Comment:
@utzig it is set here.
@jacobrosenthal I just noticed I am using lflags="-lm" on my target, when I take it out it produces the same problem.
Is this a newt bug?

Reply:
You should add `-lm` to `pkg.lflags` (not `pkg.cflags`).  However, this will only work with a fairly recent version of newt (you need this PR, merged Feb. 13: https://github.com/apache/mynewt-newt/pull/133#event-1472935366).

Reply:
BTW, this came out to be commited as cflags mistakenly. Already tried with lflags having the same problem.

Reply:
@ccollins476ad Thanks, my web browser didn't refresh this comments section so on my previous comment I wasn't aware of yours. :)